### PR TITLE
feat(collection): improve SEO

### PR DESCRIPTION
**Pull request by roiLeo**
### Before submitting this PR, please make sure:
- [x] Code builds clean without any errors or warnings
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I didn't break any original functionality

### PR type
- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring

### Changes
- fix semantic collection by adding h1
- fix galleryCard image alternative text
- add meta og:url
- add meta twitter:card
- add subscan svg alternative text

### CODE REVIEW

**Comment by yangwao** [comment]
You refer to this one https://github.com/kodadot/nft-gallery/issues/118 ?

**Comment by roiLeo** [comment]
This is not #118 related, I've added a comment in issue for telegram documentation.
This PR fix semantic issue (h1), add open graph basic Metadata (og:url) and preview twitter card for large images.

File: src/components/rmrk/Gallery/GalleryCard.vue [hunk 1]
```diff
@@ -58,9 +59,11 @@ export default class GalleryCard extends Vue {
       const meta = await get(this.metadata);
       if (meta) {
         this.image = sanitizeIpfsUrl(meta.image)
+        this.title = meta.name
```
Comment (vikiival) [inline src/components/rmrk/Gallery/GalleryCard.vue:62]: Where do you use this property?

Comment (roiLeo) [reply src/components/rmrk/Gallery/GalleryCard.vue:62]: In GalleryCard used as alternative text with the nft name

File: src/components/rmrk/Gallery/GalleryCard.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
         <b-image
           :src="image"
           :src-fallback="placeholder"
-          alt="Simple image"
+          :alt="name || 'Simple image'"
```
Comment (vikiival) [inline src/components/rmrk/Gallery/GalleryCard.vue:8]: @roiLeo you use `name` here instead of `title`

Comment (roiLeo) [reply src/components/rmrk/Gallery/GalleryCard.vue:8]: Oh nice catch

**Pull request merged**